roles/debian_physical_machine/files/ethmonitor_patch2

```diff
@@ -0,0 +1,102 @@
+@@ -59,10 +59,11 @@
+ OCF_RESKEY_pktcnt_timeout_default="5"
+ OCF_RESKEY_arping_count_default="1"
+ OCF_RESKEY_arping_timeout_default="1"
+ OCF_RESKEY_arping_cache_entries_default="5"
+ OCF_RESKEY_link_status_only_default="false"
++OCF_RESKEY_dampen_default="10s"
+ 
+ : ${OCF_RESKEY_interface=${OCF_RESKEY_interface_default}}
+ : ${OCF_RESKEY_name=${OCF_RESKEY_name_default}}
+ : ${OCF_RESKEY_multiplier=${OCF_RESKEY_multiplier_default}}
+ : ${OCF_RESKEY_repeat_count=${OCF_RESKEY_repeat_count_default}}
+@@ -70,10 +71,11 @@
+ : ${OCF_RESKEY_pktcnt_timeout=${OCF_RESKEY_pktcnt_timeout_default}}
+ : ${OCF_RESKEY_arping_count=${OCF_RESKEY_arping_count_default}}
+ : ${OCF_RESKEY_arping_timeout=${OCF_RESKEY_arping_timeout_default}}
+ : ${OCF_RESKEY_arping_cache_entries=${OCF_RESKEY_arping_cache_entries_default}}
+ : ${OCF_RESKEY_link_status_only=${OCF_RESKEY_link_status_only_default}}
++: ${OCF_RESKEY_dampen=${OCF_RESKEY_dampen_default}}
+ 
+ #######################################################################
+ 
+ meta_data() {
+ 	cat <<END
+@@ -206,10 +208,18 @@
+ </longdesc>
+ <shortdesc lang="en">link status check only</shortdesc>
+ <content type="boolean" default="${OCF_RESKEY_link_status_only_default}" />
+ </parameter>
+ 
++<parameter name="dampen" reloadable="1">
++<longdesc lang="en">
++The time to wait (dampening) further changes occur
++</longdesc>
++<shortdesc lang="en">Dampening interval</shortdesc>
++<content type="integer" default="10s"/>
++</parameter>
++
+ </parameters>
+ <actions>
+ <action name="start" timeout="60s" />
+ <action name="stop" timeout="20s" />
+ <action name="status" depth="0" timeout="60s" interval="10s" />
+@@ -272,10 +282,13 @@
+ 				ocf_exit_reason "Interface $NIC does not exist"
+ 				exit $OCF_ERR_CONFIGURED;;
+ 			monitor)
+ 				ocf_log debug "Interface $NIC does not exist"
+ 				;;
++			monitor)
++				ocf_log debug "Interface $NIC does not exist"
++				;;
```

Comment:
monitor action twice?=> I guess either merge the 2 patches or remove this hunk in this one.